fineract-provider/src/main/resources/db/changelog/tenant/custom-changelog/INKO-250_loan_document_type.xml

```diff
@@ -35,9 +34,16 @@
         </addColumn>
     </changeSet>
     <changeSet author="tabrez@fiter.io" id="add_document_type_code">
```

Comment:
@tabrez-fiter  It looks like this changeset already ran on dev server. Please change the id name to avoid checksum issues/ breaking the application startup